_releases/2023-04-12-1.8.0-released.md

```diff
@@ -0,0 +1,137 @@
+
+### AArch64 Android
+
+It is now possible to target Android with the Bionic C runtime (`aarch64-linux-android`). The details of how to get this working are in the relevant PR ([#13065](https://github.com/crystal-lang/crystal/pull/13065)).
+
+### Windows
+
+We're making progress on reaching full stdlib feature coverage for Windows.
+
+This release includes a couple of improvements relating to symbolic links ([#13141](https://github.com/crystal-lang/crystal/pull/13141), [#13195](https://github.com/crystal-lang/crystal/pull/13195))
+⚠️ An important change to align behaviour with POSIX platforms is that `File.delete` (instead of `Dir.delete`) removes *any* symlink, even when it points to a directory ([#13224](https://github.com/crystal-lang/crystal/pull/13224)).
+
+OpenSSL now uses Windows' system root certificate store ([#13187](https://github.com/crystal-lang/crystal/pull/13187)).
+
+⚠️ This release also adds a full stub for Windows signals ([#13131](https://github.com/crystal-lang/crystal/pull/13131)). More on that in the next section.
+
+While Windows 7 has reached EOL, we understand that support is still necessary for some use cases. Therefore, we accepted a patch to lower the supported Windows version ([#11505](https://github.com/crystal-lang/crystal/pull/11505)).
+
+## Signal
+
+In Unix systems, inter-process communication such as terminating processes and responding to interrupts is based on signals and subsequently the `Signal` API in Crystal was used for this.
+To support operating systems that use different mechanisms for that (such as Windows), it's necessary to have portable APIs which abstract the intention from the implementation.
+
+* [`Process.on_interrupt`](https://crystal-lang.org/api/1.8/Process.html#on_interrupt%28%26handler%3A-%3E%29%3ANil-class-method) handles interrupts: `SIGINT` (Unix) and `Ctrl+C` or `Ctrl+Break` (Windows) ([#13034](https://github.com/crystal-lang/crystal/pull/13034)).
+* [`Process#terminate`](https://crystal-lang.org/api/1.8/Process.html#terminate%28%2A%2Cgraceful%3ABool%3Dtrue%29%3ANil-instance-method) gained a `graceful` parameter. On Unix it selects between `SIGTERM` and `SIGKILL` while it has no effect on Windows and the behaviour is similar to `taskkill /f` regardless ([#13070](https://github.com/crystal-lang/crystal/pull/13070)).
+* [`Process::Status#exit_reason`](https://crystal-lang.org/api/1.8/Process/Status.html#exit_reason%3AExitReason-instance-method) and [`Process::ExitReason`](https://crystal-lang.org/api/1.8/Process/ExitReason.html) provide information about a program's exit status in a portable way ([#13052](https://github.com/crystal-lang/crystal/pull/13052)).
+
+## Stdlib
+
+We fixed some function definition for `libc` that used the wrong types (e.g. `Int` instead of `SizeT`) in [#13242](https://github.com/crystal-lang/crystal/pull/13242) and [#13249](https://github.com/crystal-lang/crystal/pull/13249). This affects a bug with sending huge network packets on `*-linux-gnu` targets.
```

Comment:
```suggestion
We fixed some function definitions for `LibC` that used the wrong types (e.g. `Int` instead of `SizeT`) in [#13242](https://github.com/crystal-lang/crystal/pull/13242) and [#13249](https://github.com/crystal-lang/crystal/pull/13249). This affects a bug with sending huge network packets on `*-linux-gnu` targets.
```